apps/api/src/modules/notifications/notifications.service.ts

```diff
@@ -36,11 +43,60 @@ export class NotificationsService {
       throw new BadRequestException(`Channel ${notification.channelType} is not enabled`);
     }
 
-    notification.createdBy = this.configService.getOrThrow<string>('APP_NAME') || 'OsmoX';
-    notification.updatedBy = this.configService.getOrThrow<string>('APP_NAME') || 'OsmoX';
+    // Set correct ApplicationId
+    notification.applicationId = await this.getApplicationIdFromApiKey(authHeader);
+
+    // Set correct application name using applicationId
+    notification.createdBy = 'OsmoX';
```

Comment:
Created By value should also be fetched from the application ID only.

Reply:
Ok